### ISSUE TITLE
[Suggestion] Per-nutrient Multiplier

### ISSUE THREAD

**Issue by Alessandro404** [WesCook/Nutrition#108]
Right now every food with multiple nutrients split evenly Its value, and every nutrients gauge fills at the same speed.

Before setting single items with different nutrition %s, maybe we can have a modifier for the gauge size, so one could make a some  bars fills faster or slower than others. Like proteins vs carbohydrates, eating food that cointains both.

**Comment by WesCook** [comment]
Hi @Alessandro404,

Thanks for for your suggestion.  However having foods share nutrition values equally was a design choice to reduce overall complexity.  I feel that having to specify both nutrients and percentage values of each would require significantly more maintenance when building a pack, and would require more information to communicate to the end player.  So unfortunately this is not a feature I plan to implement.

Thank you,
Wes

**Comment by Alessandro404** [comment]
Thank you for your answer. 

I want to point however that what I'm asking would not require any major manteinnance from a modpack maker standpoint. It would be a simple multiplier inside each nutrition type, and that would remain identical for every food. 

Right now I am trying to build a modpack, so I would appreciate more flexibility without having to use fake nutrion bars (like double for proteins) or craftweaker to launch a command for setting custom  values every time a food is ingested :D.
my advice is to let the modmakers decide whether to make the system more complex.

**Comment by WesCook** [comment]
Hi,

Thanks for the further explanation.  Sorry about that, I misunderstood your first post.

A nutrient multiplier is actually something that would make for a good feature.  I was surprised it didn't exist yet, as it should be fairly simple to add.  I can't give an ETA but I'll look into adding this the next time I do some dev work.

**Comment by Alessandro404** [comment]
I'm glad I was able to explain my suggestion!
Thank you for your time

**Comment by WesCook** [comment]
I looked into this feature today.  I realized that the current design of the mod doesn't allow this right now as it's assumed that all nutrients are of equal value.  For instance the tooltip will show the calculated value shared by all nutrients.  It would be possible, but more complicated to separate that into showing multiple nutritional values for different food groups.

So unfortunately despite my initially believing this is possible, it now looks like it is not.  I'm sorry if you were depending on this feature.  Thank you though for the suggestion regardless.
